packages/ag-charts-community/src/chart/cartesianUtil.ts

```diff
@@ -0,0 +1,39 @@
+import { CartesianSeries } from './series/cartesian/cartesianSeries';
+import type { ISeries } from './series/seriesTypes';
+
+export function stackCartesianSeries(series: ISeries<unknown, unknown, unknown>[]) {
+    const seriesGroups = new Map<string, CartesianSeries<any, any, any, any, any, any, any>[]>();
```

Comment:
Nit: Can we use `unknown` rather than `any` by default?

Reply:
I was able to change one of these